pennylane/transforms/__init__.py

```diff
@@ -71,6 +71,8 @@
     :toctree: api
 
     ~transforms.zyz_decomposition
+    ~transforms.get_unitary_matrix
```

Comment:
Is that really used as a compilation utility tool? Maybe it would fit better in transforms that act on qnodes?

Reply:
Yeah, where to put it was basically my [question](https://github.com/PennyLaneAI/pennylane/pull/1609#issuecomment-915050584) a couple of days ago. I wasn't sure where to put it, since it acts on qnodes, tapes, _and_ quantum functions. For that reason it doesn't feel quite right to put it under only one of them, but putting it under all three would be a bit excessive perhaps... Another option would be to make a new section for it. What do you think would be best?

Reply:
I spoke with @glassnotes and we think that it would be better to have it in the qnode section as it will be the main user facing part. But you can add in the documentation examples that show the other utilities of the transform like with a tape👍